Frends.MicrosoftSQL.ExecuteQuery/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.2.2] - 2024-06-03
+### Changed
+- Changed the library from System.Data.SqlClient to Microsoft.Data.SqlClient 
+- Added environment variables for secrets
```

Comment:
This change (if it goes through) is not really needed to be in the public changelog, as it is not affecting task users.